shared/src/persistence/elasticsearch/getReadyForTrialCases.js

```diff
@@ -39,7 +34,7 @@ exports.getReadyForTrialCases = async ({ applicationContext }) => {
               },
               {
                 range: {
-                  'servedAt.S': {
+                  'createdAt.S': {
```

Comment:
Does this need to be added to `efcms-docket-entry-mappings.js`?

Reply:
Hmm. Good catch.